groups/sig-cluster-lifecycle/groups.yaml

```diff
@@ -116,8 +116,11 @@ groups:
     settings:
       ReconcileMembers: "true"
     members:
```

Comment:
@mdbooth I think you will want to update that one too: https://github.com/kubernetes/k8s.io/blob/main/groups/sig-cluster-lifecycle/groups.yaml#L120